### ISSUE TITLE
Muse Sounds oboe E5 - A4 transition skips or delays the A4 

### ISSUE THREAD

**Issue by MarcSabatella** [musescore/MuseScore#22888]
### Issue type

Muse Sounds bug

### Bug description

If you write E5 - A4 for oboe, the A4 is played very late if at all.  If there is no subsequent note, the A4 is not played at all.  If there is a subsequent note, the A4 is played almost as a grace note into the next note.

### Steps to reproduce

1. New score for oboe
2. Enter half notes: E5 A4 C5 E5 A4
3. Play

Result: first A4 is played very late and almost like a grace note; second A4 is not played at all

Problem seems to occur only at tempos > 120, and only at mp and mf dynamics.  It goes away when adding sound flags or articulations etc.

### Screenshots/Screen recordings

[Screen recording 2024-05-17 7.56.37 PM.webm](https://github.com/musescore/MuseScore/assets/1799936/a1c8f327-f4b9-4740-9076-e727783e56ed)

### MuseScore Version

OS: Debian GNU/Linux 12 (bookworm), Arch.: x86_64, MuseScore Studio version (64-bit): 4.3.0-241231431, revision: github-musescore-musescore-5f36e74

### Regression

I don't know

### Operating system

Debian 12

### Additional context

Using Muse Woodwinds 0.5.20 and Muse Sampler 0.6.3.196

**Comment by wizofaus** [comment]
Yep, very straight forward to reproduce, and not dissimilar to other weird artifacts I hear regularly where the rhythm is thrown out by notes only becoming audible just before the next note starts.
